Sources/Apodini/Application/Application+Connection.swift

```diff
@@ -1,21 +1,18 @@
 //
 //  Connection.swift
-//  
+//
 //
 //  Created by Moritz Schüll on 09.12.20.
 //
 
 import Foundation
 
-enum ConnectionEnvironmentKey: EnvironmentKey {
-    static var defaultValue = Connection()
-}
-
-extension EnvironmentValues {
+extension Application {
     /// A Property identifying the `Connection` which provides an abstract view on the underlying protocol's state.
+    /// - Note: This property only serves as a placeholder and should never be called.
```

Comment:
```suggestion
    ///
    /// This property only serves as a placeholder and should never be called.
```

If you make a break (leave one line blank), Xcode will already display this section below a heading _Discussion_